scrapy/utils/conf.py

```diff
@@ -99,14 +99,20 @@ def init_env(project="default", set_syspath=True):
             sys.path.append(projdir)
 
 
-def get_config(use_closest=True):
```

Comment:
Is it a general refactoring, or is it needed for the PR to work?

Reply:
It's no longer needed.